app/assets/stylesheets/application/contributors.scss

```diff
@@ -1,9 +1,90 @@
-.hideContent {
-  display: none;
-}
+.contributorModalBody {
+	display: flex;
+	flex-direction: row;
+
+	.contributorModalBodyPhoto {
+    flex: 0 1 30%;
+
+    &.showOnSmall {
+      display: none;
+    }
+
+    .contributorAvatar {
+      border-radius: 50%;
+      overflow: hidden;
+      object-fit: cover;
+      background: linear-gradient(104.26deg, #6d0839 0%, #D0E799 175.81%);
```

Comment:
I think you should be able to use `$mulberry` and `$key-lime` for the respective hex values.